app/controllers/tasks_controller.rb

```diff
@@ -23,6 +23,7 @@ def create
   def start
     authorize! :start, @task
     @task.start!
+    @task.update_attribute(:started_at, Time.now.to_datetime)
```

Comment:
I think this line can be moved to the `start!` method called above, so that `start!` will update both fields `status` and `started_at`